spynl/main/__init__.py

```diff
@@ -1,17 +1,25 @@
 """The main package of Spynl."""
 
-
-import os.path
+from pkg_resources import iter_entry_points  # pylint: disable=E0611
 
 from pyramid.config import Configurator
-from pyramid.settings import asbool
 from pyramid.security import NO_PERMISSION_REQUIRED
 from pyramid.viewderivers import INGRESS
 
-from spynl.main import (plugins, serial, about,
-                        routing, events, endpoints, session)
-from spynl.main.utils import (renderer_factory, check_origin,
-                              handle_pre_flight_request)
+from spynl.main import (
+    serial,
+    about,
+    plugins,
+    routing,
+    events,
+    endpoints,
+    session,
+)
+from spynl.main.utils import (
+    renderer_factory,
+    check_origin,
+    handle_pre_flight_request,
+)
```

Comment:
Personally, I would prefer to keep this as it was, because we use the same format everywhere. (where the split up is can of course be different because it looks as if routing should fit on the line above)